### PR TITLE
refactor: Export table list generation

### PR DESCRIPTION
**Pull request by bernielomax**
I would like to be able to use the same logic in other external projects.

### CODE REVIEW

**Review by svysali — APPROVED** [review]
Just the one change, but otherwise looks good to me.

File: db/schema.go [hunk 1]
```diff
@@ -154,35 +149,51 @@ func createTriggerFunc(tx *pgx.Tx) error {
...
+				return nil, err
 			}
 
-			if _, ok := exclude[t]; !ok {
-				tables = append(tables, fmt.Sprintf("%s.%s", schema, t))
+			for rows.Next() {
+				var t string
+				err = rows.Scan(&t)
+				if err != nil {
+					return nil, nil
```
Comment (svysali) [inline db/schema.go]: this needs to be `return nil, err`

**Review by svysali — CHANGES_REQUESTED** [review]

File: db/schema.go (hunk 1 above)
Comment (bernielomax) [reply db/schema.go]: ah nice! thanks. that was a copy and paste 😂

Comment (bernielomax) [reply db/schema.go]: done

**Review by svysali — APPROVED** [review]

**Pull request merged**